tap_spreadsheets_anywhere/__init__.py

```diff
@@ -107,21 +107,26 @@ def sync(config, state, catalog):
                 key_properties=stream.key_properties,
             )
 
-            # Emit a Singer ACTIVATE_VERSION message before initial sync (but not subsequent syncs)
-            # and everytime a sheet sync is complete.
-            # This forces hard deletes on the data downstream if fewer records are sent.
-            # https://github.com/singer-io/singer-python/blob/master/singer/messages.py#L137
-
-            is_initial_sync = 'modified_since' not in state.get(stream.tap_stream_id, {})
-            activate_version = int(time.time() * 1000)
-            activate_version_message = singer.ActivateVersionMessage(
-                stream=stream.tap_stream_id,
-                version=activate_version,
-            )
-            if is_initial_sync:
-                # initial load, send ACTIVATE_VERSION message before the data sync
-                singer.write_message(activate_version_message)
-                LOGGER.info(f'INITIAL SYNC, Stream: {stream.tap_stream_id}, Activate Version: {activate_version}')
+            full_table_replace = table_spec['full_table_replace'] if 'full_table_replace' in table_spec else False
```

Comment:
- [ ] Could this be `full_table_replace = table_spec.get('full_table_replace', False)`?

Reply:
Yes definitely 👍 Changed now.